NamuLink.user.js

```diff
@@ -17,7 +17,7 @@
 // @exclude      https://namu.wiki/w/member/recover_password*
 // @exclude      https://namu.wiki/w/member/signup*
 //
-// @description        NamuLink blocks the license-abused PowerLink advertisement on NamuWiki.
+// @description  NamuLink blocks the license-abused PowerLink advertisement on NamuWiki.
```

Comment:
This can be used when providing multi-language description.
https://www.tampermonkey.net/documentation.php#_description